server/services/hemocioneIdFacts.ts

```
@@ -0,0 +1,32 @@
+// Deliberately catches internally — called via runAsync/waitUntil with no caller
+// awaiting the result, so a rejected promise here would only surface as an
+// unhandled rejection instead of failing anything meaningful.
+export async function postFactToHemocioneId(fact: GamificationFact) {
+  try {
+    const response = await fetch(`${config.hemocioneIdApiUrl}/internal/facts`, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        "x-secret": config.hemocioneIdFactsSecret,
+      },
+      body: JSON.stringify({ ...fact, sourceService: "hemocione-digital-event" }),
```

ESLint

[error] Replace `·...fact,·sourceService:·"hemocione-digital-event"` with `⏎········...fact,⏎········sourceService:·"hemocione-digital-event",⏎·····`

(prettier/prettier)